reports/report_builder.py

```diff
@@ -1343,6 +1348,9 @@ def ws_32(self, ws):
         secondary_counts = OrderedDict()
 
         for media_type, model in journalist_models.items():
+            if media_type == 'Twitter':
+                # Skip twitter reporters
+                continue
```

Comment:
Wasn't this like the perfect opportunity to use [Generator Expression](http://www.python.org/dev/peps/pep-0289/)?

```python
journalist_models_items = (media_type, model for media_type, model in journalist_models.items() if model_type != "Twitter")
for media_type, model in journalist_models_items:
     ....
```

or something like that (haven't tested the code).